packages.json

```diff
@@ -7,73 +7,73 @@
   },
   {
     "name": "@kadena/chainweb-node-client",
-    "version": "0.4.4",
+    "version": "0.5.0",
```

Comment:
No idea why this changes 🤷  these changes happened after running `pnpm format`..

Reply:
Yes they should be changed when [ci] Release PR is merged. @Ghislain89 is looking into it